lib/chef/knife/oci_server_delete.rb

```diff
@@ -82,6 +83,11 @@ def delete_chef_node(node)
         ui.msg "Deleted Chef node '#{node.name}'"
       end
 
+      def delete_chef_client(client_name)
```

Comment:
Should include a message as in delete_chef_node (unless something is being printed by the all already being made?).

Reply:
The most recent update to this pull request does issue a message when
deleting the 'client' from Chef.  (There was already a message for the 'node'
deletion.)